homeassistant/components/fritz/services.py

```diff
@@ -1,21 +1,21 @@
 """Services for Fritz integration."""
 
-from __future__ import annotations
-
 import logging
 
 import voluptuous as vol
 
-from homeassistant.config_entries import ConfigEntryState
 from homeassistant.core import HomeAssistant, ServiceCall
 from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers import config_validation as cv
 from homeassistant.helpers.service import async_extract_config_entry_ids
 
-from .const import DOMAIN, FRITZ_SERVICES, SERVICE_SET_GUEST_WIFI_PW
+from .const import DOMAIN, SERVICE_SET_GUEST_WIFI_PW
 from .coordinator import AvmWrapper
 
 _LOGGER = logging.getLogger(__name__)
 
+CONFIG_SCHEMA = cv.config_entry_only_config_schema(DOMAIN)
```

Comment:
Is this needed here?

Reply:
yes, hassfest comlains about it

Reply:
nevermid ... it does not complain anymore, nit sure why it did in the past (_2 days ago_)

Reply:
```
Error: R] [CONFIG_SCHEMA] Integrations which implement 'async_setup' or 'setup' must define either 'CONFIG_SCHEMA', 'PLATFORM_SCHEMA' or 'PLATFORM_SCHEMA_BASE'. If the integration has no configuration parameters, can only be set up from platforms or can only be set up from config entries, one of the helpers cv.empty_config_schema, cv.platform_only_config_schema or cv.config_entry_only_config_schema can be used.
```

its back :grimacing: I think i've to rebuilt my dev-container :see_no_evil:

Reply:
It's needed in __init__.py
But not in services.py

Reply:
:facepalm: you're right ... we're in the `services.py` not in the `__init__.py`